### PR TITLE
Layer creation frontend

### PR DESCRIPTION
**Pull request by nbuonin**
This PR implements endpoints for layer CRUD ops, and roughs in some interface to make it go.

### CODE REVIEW

File: media/src/project/project-map.tsx [hunk 1]
```diff
@@ -32,28 +38,95 @@ export const ProjectMap = () => {
...
 
-    const [
-        staticMapViewport,
-        setStaticMapViewport
-    ] = React.useState(viewportState);
+    const deleteLayer = (pk: number) => {
+        authedFetch(`/api/layer/${pk}/`, 'DELETE', JSON.stringify({pk: pk}))
+            .then((response) => {
+                if (response.status !== 204) {
+                    throw 'Layer creation failed.';
```
Comment (sdreher) [inline media/src/project/project-map.tsx:98]: layer deletion failed?

File: media/src/project/project-map.tsx [hunk 2]
```diff
@@ -32,28 +38,95 @@ export const ProjectMap = () => {
...
+    };
+
+    const updateLayer = (pk: number, title: string) => {
+        authedFetch(`/api/layer/${pk}/`, 'PUT', JSON.stringify(
+            {title: title, content_object: `/api/project/${projectPk}/`}))
+            .then((response) => {
+                if (response.status === 200) {
+                    return response.json();
+                } else {
+                    throw 'Layer creation failed.';
```
Comment (sdreher) [inline media/src/project/project-map.tsx:114]: layer update failed?

File: media/src/project/project-map.tsx (hunk 1 above)
Comment (nbuonin) [reply media/src/project/project-map.tsx]: 👍

File: media/src/project/project-map.tsx (hunk 2 above)
Comment (nbuonin) [reply media/src/project/project-map.tsx]: 👍

**Pull request merged**